solutions/swb-app/src/dataSets/dataSetAccessLevelParser.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const DataSetAccessLevelParser = z.union([z.literal('read-only'), z.literal('read-write')]);
```

Comment:
[](https://media.giphy.com/media/mZGV0HkpYK3y8/giphy.gif)

Reply:
Was there a comment here?

Reply:
ugh, github failure to display a gif of Paris Hilton saying "Thats hot"